modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -207,11 +207,9 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
                          );
         $this->columns = array_merge(
             $this->columns,
-            array(
-             "GROUP_CONCAT(DISTINCT OutputType) as Links",
-             's.ID as sessionID',
-            ),
-            $modalities_subquery_as
+            array("GROUP_CONCAT(DISTINCT OutputType) as Links"),
+            $modalities_subquery_as,
+            array('s.ID as sessionID')
```

Comment:
this change is necessary to make the order match that of the `$this->headers` where `sessionID` should come in the end